src/box/box.cc

```diff
@@ -4679,6 +4709,17 @@ box_process_subscribe(struct iostream *io, const struct xrow_header *header)
 	if (replica == NULL)
 		replica = replicaset_add_anon(&req.instance_uuid);
 
+	if (!req.is_anon && box_gc_consumer_is_persistent() &&
+	    replica->gc == NULL) {
+		tnt_raise(ClientError, ER_PROTOCOL, "Can't subscribe "
+			  "non-anonymous replica without gc_consumer");
+	}
+
+	if (replica->gc != NULL && replica->gc->with_snap) {
```

Comment:
Should be in another patch, stubs for EE persistent GC

Reply:
Dropped the snippet.